apps/server/src/tasks/entities/chunkInfo.entity.ts

```diff
@@ -1,33 +1,81 @@
-import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import moment, { Duration } from 'moment';
 import { Field, ObjectType } from '@nestjs/graphql';
-import { Duration } from 'moment';
 import { IntervalColumn } from '../../types/IntervalColumn';
 import { GeneralEntity } from '../../utils/GeneralEntity';
+import { NullableColumn } from '../../utils/NullableColumn';
+import { NullableField } from '../../utils/NullableField';
 import { Interval } from '../../utils/interval.scalar';
+import { Repeat } from './repeat.entity';
 
+// TODO Those fields need to be virtual and better-organized.
 @Entity()
 @ObjectType()
 export class ChunkInfo extends GeneralEntity {
   @Field()
   @PrimaryGeneratedColumn()
   id: number;
 
+  // Shared properties
   @Field({
     description:
-      'The time when task should start. This can be different from taskBreakdown.start, as it is just informative data unrelated with real planed entity.',
+      'The time when task should start. In case of float tasks this can be different from taskBreakdown.start, as it is just informative data unrelated with real planed entity.',
   })
   @Column({ type: 'timestamp with time zone' })
   start: Date;
 
-  @Field(() => Interval)
+  @Field(() => Interval, {
+    // 15 minutes is a default value
+    defaultValue: moment.duration(15, 'minutes').toISOString(),
+    description:
+      'A minimum time gap that user wants to have between this task and another tasks.',
+  })
   @IntervalColumn()
-  minTimeBetweenChunks: Duration;
+  chillTime: Duration;
```

Comment:
as I understand for float task we no longer want to differentiate chillTime and minTimeBetweenChunks? I think that's fine since in task addition form those two fields next to each other were confusing, but I believe that we should think about some mechanism that would protect form planning multiple chunks of the same task for 1 day. What do You think?

Reply:
I think that `minTimeBetweenChunks` field should really be removed - it complicates the algorithm and reduces the chance that the task can be scheduled.
Agree with @dimamik about chillTime and planning multiple chunks on the same day. Let`s keep our algorithm as simple as possible for now, in case we will have time we can always make it more complicated.